Cogs/Gourmet.py

```diff
@@ -0,0 +1,108 @@
+        async def add(self,interaction:discord.Interaction,button:discord.ui.Button):
+
+            # This must be here because of the user response
+            # If the user response is before this it would show "interaction failed"
+            # The interaction has not really failed however it hasnt happened quick enough.
+            await interaction.response.send_message(f'Enter a restaurant to add to the list.')
+
+            # Waiting for the user's response (with interactions!)
+            msg = await interaction.client.wait_for('message', check=lambda message: message.author == interaction.user)
+
+            # Turning the message to lowercase and appending it
+            msg.content = msg.content.casefold()
+            for rest in self.cog.normal_restaurant:
+                if msg.content == rest.casefold():
+                    await interaction.message.reply(f'Error: {msg.content} already is in the restaurant list.')
+                    return
+            self.cog.normal_restaurant.append(msg.content)
+
+        # Removes a restaurant from a list
+        @discord.ui.button(label="Remove Restaurant",style=discord.ButtonStyle.red)
+        async def remove(self,interaction:discord.Interaction,button:discord.ui.Button):
+
+            # This must be here because of the user response
+            # If the user response is before this it would show "interaction failed"
+            # The interaction has not really failed however it hasnt happened quick enough.
+            await interaction.response.send_message(content=f'Enter a restaurant to remove from the list.')
+
+            # Waiting for the user's response (with interactons!)
+            msg = await interaction.client.wait_for('message', check=lambda message: message.author == interaction.user)
+
+            # Turning msg to lowercase
+            msg.content = msg.content.casefold()
+
+            # If the restaurant is not in the list it responds with an error
+            for rest in self.cog.normal_restaurant:
+                if msg.content == rest.casefold():
+                    self.cog.normal_restaurant.remove(rest)
+                    return
+            await interaction.message.reply(content=f'Error: {msg.content} is not in the list.')
+        
+        # Returns a random vegan restaurant
+        @discord.ui.button(label="Vegan",style=discord.ButtonStyle.blurple,emoji='\U0001F96C')
+        async def vegan(self,interaction:discord.Interaction,button:discord.ui.Button):
+            if not len(self.cog.vegan_restaurant) == 0:
+                await interaction.response.send_message(self.cog.vegan_restaurant.pop())
+            else:
+                await interaction.response.send_message('You have run out of restaurants. You are going to arbys.')
+
+    @commands.command()
+    @commands.has_permissions(administrator=True)
+    async def feedme(self, ctx):
+        self.normal_restaurant = []
```

Comment:
I think a cleaner implementation would be to find a way to initialize these lists when the Cog is loaded.

Talk with Anmol about the work he's doing with faqbot. We read the file in the setup() function, and then pass some additional parameters to the Cog Initialization.

Once you have the file read in, feedme can work by making copies of the attributes:
`restaurants = self.normal_restaurants`